.github/workflows/pages.yml

```diff
@@ -4,8 +4,6 @@ on:
   push:
     branches:
     - master
-    tags:
```

Comment:
tags should be building the docs ?

Reply:
All versions listed in ``conf.py`` are already recreated every time the master branch receives a new commit, so there is no need to do so as well every time a new version is tagged.

This is to prevent the failure we had the other day. If the latest branch had one configuration, but the tagged version another (e.g. we are not tagging the latest commit), all docs were being built with an incorrect configuration file.